Add files via upload

Nabefumi opened the pull request:
・Add change background color of each text field inside LoginViewController and SignUpViewController
・Add pass title of LoginViewController and SignUpViewController using segue and from ViewController
・Add Observe changes on textFields and validate input:
a. username length should be between 6 and 14
b. email should contain @  character always
c. password length should be between 8 and 20

Nabefumi committed:
Add files via upload

・Add change background color of each text field inside LoginViewController and SignUpViewController
・Add pass title of LoginViewController and SignUpViewController using segue and from ViewController
・Add Observe changes on textFields and validate input:
a. username length should be between 6 and 14
b. email should contain @  character always
c. password length should be between 8 and 20

Mostafa-Davoodi commented:
@Nabefumi please try to use a meaningful commit messages.

Mostafa-Davoodi commented:
@Nabefumi structure of your project is wrong since you should have your project at the root directory, not in the child directory. please move your file to the parent directory.

Nabefumi committed:
Add functions and Change the directory structure

Add functions and Change the directory structure of Project

[Fix]  Change the directory structure of Project

--
[Add] toLoginViewController.swift
[Add] toSignUpViewController.swift

--
[Update] Change the background color of the TextField: Editing mode: GREEN and Otherwise WHITE
[Update]  Pass LoginViewController and SignUpViewController using segue and from WiewController
[Update]  userNameTextFieldinput Input check: Up to 14 characters and change the background color to red when the input characters are 6 or less
[Update]  emailTextField: Input check: @  character always and display an error message if it is not in mail format
[Update]  passwordTextField Input check: Up to 20 characters
[Update]  Pass the input value entered in SignUpViewController to the next controller toSignUpViewController
[Update]  Pass the input value entered in LoginViewController to the next controller, toLoginViewController.

Nabefumi commented:
> please try to use a meaningful commit messages.

I fixed the commit Comment.

> structure of your project is wrong since you should have your project at the root directory, not in the child directory. please move your file to the parent directory.

I fixed the structure of directory.
